src/components/operate.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-undef */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+  sum, subtract, mutiply, devide,
+} from './operate';
+
+import ButtonBox from './ButtonBox';
+import About from './Pages/About';
+import Quate from './Pages/Quate';
+import CalcProvider from '../context/CalcContext';
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
+
+test('adds 4 - 2 to equal 2', () => {
+  expect(subtract(4, 2)).toBe(2);
+});
+
+test('adds 4 * 2 to equal 2', () => {
+  expect(mutiply(4, 2)).toBe(8);
+});
+
+test('adds 4 / 2 to equal 2', () => {
+  expect(devide(4, 2)).toBe(2);
+});
+
+describe('ButtonBox', () => {
+  test('Renders ButtonBox component', () => {
+    render(<ButtonBox />);
+    screen.debug();
+  });
+});
+
+describe('About', () => {
+  test('Renders About component', () => {
+    render(<About />);
+    screen.debug();
+  });
+});
+
+describe('Quate', () => {
+  test('Renders Quate component', () => {
+    render(<Quate />);
+    screen.debug();
+  });
+});
+
+describe('CalcProvider', () => {
+  test('Renders CalcProvider component', () => {
+    render(<CalcProvider />);
+    screen.debug();
+  });
+});
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md) for this project, " `Run your full tests suite using Jest. All tests should pass` ", please make sure your tests are running, I am unable to run your tests because of the error in the screenshot, please checkout the screenshot and kindly fix the error.

![mathTestError](https://user-images.githubusercontent.com/65852621/185549736-f8505593-9d52-4ae8-875d-ca5a5e64448a.png)

Reply:
Hi, @Mihndim2020, thank you for the nice review. Please see the screenshot, I am running the test, and all tests passed successfully. I have installed the jest test and the React Library locally on my computer by using:
npm install --save-dev @types/jest  
and 
npm install --save-dev @testing-library/react
<img width="465" alt="tests" src="https://user-images.githubusercontent.com/22744775/185562020-a2945495-a63d-4cff-b753-fcd1b042d99e.PNG">